Cargo.toml

```diff
@@ -172,20 +167,7 @@ rand = []
 runtime = []
 
 # Enable all API features.
-all-apis = [
-    "fs",
-    "io_uring",
-    "mm",
-    "net",
-    "param",
-    "process",
-    "procfs",
-    "rand",
-    "runtime",
-    "termios",
-    "thread",
-    "time",
-]
+all-apis = ["fs", "io_uring", "mm", "net", "param", "process", "procfs", "rand", "runtime", "termios", "thread", "time"]
```

Comment:
Similarly, this creates a very long line, and this line changes over time, and keeping each feature on its own line makes it easier to read diffs.